Revert "Update mimecast-source.md (#4510)" (#5106)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/mimecast-source.md

### Vendor configuration

> The integration must be configured with the Client ID and Secret Key. Also, the user account associated with your Mimecast credentials needs to have `basic administrator` access.

Reviewer: ```suggestion
To configure the integration, you must use a Client ID and Secret Key. Additionally, the user account associated with your Mimecast credentials must have `basic administrator` access.
```

> Refer to the [Mimecast documentation](https://developer.services.mimecast.com/api-overview#application-registration-credential-management) for guidance to create the Client ID and Secret Key.

Reviewer: ```suggestion
Refer to the [Mimecast documentation](https://developer.services.mimecast.com/api-overview#application-registration-credential-management) for guidance to create the Client ID and Client Secret key.
```
